### DISCUSSION TITLE
Perimeta SBC

### DISCUSSION THREAD

**Discussion by lblyden** [sipcapture/homer#564] [Q&A]
We have Perimeta SBCs in our environment. Has anyone had any luck with this platform?

**Comment by adubovikov** [comment]
Perimeta SBC is Metaswitch and they should provide somehow the logs and metrics information. Just ask them and you can use pastash to inject the data to Homer

**Comment by lblyden** [comment]
Thank you Alexandr.

We have full access to our SBCs, we just don't know which logs and metrics
are needed to inject into Homer.

**Reply by adubovikov** [reply]
ah, this is even easy, what is OS inside ? Debian base ? Can you share more information like: kernel version, libc version etc... 
do you have root access or just cli ?

**Comment by vincenthorst** [comment]
Normally with Perimeta you have only the CDR logs.
There is no forwarded for other details.
Rest of the details you get from the analytics out of the SAS server. But the default forward there is Splunk.
________________________________
Van: Alexandr Dubovikov ***@***.***>
Verzonden: Thursday, October 20, 2022 11:30:30 PM
Aan: sipcapture/homer ***@***.***>
CC: Subscribed ***@***.***>
Onderwerp: Re: [sipcapture/homer] Perimeta SBC (Discussion #564)


ah, this is even easy, what is OS inside ? Debian base ? Can you share more information like: kernel version, libc version etc...
do you have root access or just cli ?

**Comment by lmangani** [comment]
@vincenthorst swap splunk with our new stack [qryn](https://qryn.dev) to save millions of dollars and minutes

**Comment by adubovikov** [comment]
looks like a good way for you @lblyden to use qryn as @lmangani suggested
